### PR TITLE
repositories: Add robertgzr's overlay

### PR DESCRIPTION
**Pull request by robertgzr**

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5644,5 +5644,17 @@
     <source type="git">git@github.com:kabili207/zyrenth-overlay.git</source>
     <feed>https://github.com/kabili207/zyrenth-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>robertgzr</name>
+    <description lang="en">robertgzr's personal overlay.</description>
+    <homepage>https://git.sr.ht/~robertgzr/portage</homepage>
+    <owner type="person">
+      <email>r@gnzler.io</email>
```
Comment (mgorny) [inline files/overlays/repositories.xml:5652]: I don't seem to be able to find such e-mail on Gentoo Bugzilla. Are you registered? We require the contact address to correspond to a Bugzilla account, so we can assign bugs reported by users.

Comment (robertgzr) [reply files/overlays/repositories.xml:5652]: ahh does that email have to match... my bugzilla account uses `robert@gnzler.io`

Comment (mgorny) [reply files/overlays/repositories.xml:5652]: Yes, please update either the PR or Bugzilla, and let me know once done.

Comment (robertgzr) [reply files/overlays/repositories.xml]: @mgorny updated

**Review by mgorny — APPROVED** [review]
Thanks!

**Comment by robertgzr** [comment]
@mgorny thanks o/ I didn't realize the `<repositories>` object would be taken as-is. I should've probably sorted my entry in alphabetically looking at overlays.gentoo.org

**Pull request closed without merging**